src/Formatter/PlatesFormatter.php

```diff
@@ -22,23 +26,25 @@ public function __construct(Engine $engine)
     }
 
     /**
-     * @inheritDoc
+     * Get a copy that uses a different template.
+     *
+     * @param string $template
+     *
+     * @return static
      */
-    public function body(PayloadInterface $payload)
+    public function withTemplate($template)
```

Comment:
Since the `template` value can no longer be passed with payload "settings" this was necessary to allow setting the template before rendering. Seems straight forward enough (see test case).